frontend/src/components/dashboard/campaigns/announcement-modal/AnnouncementModal.module.scss

```diff
@@ -64,7 +74,13 @@ $title-height: $height-4;
 }
 
 .options {
+  margin-top: 25px;
+
   @include mobile {
     padding-bottom: 25px;
   }
+
+  & > *:not(:first-child) {
+    margin-left: $spacing-7;
```

Comment:
I've changed this from 2rem to `$spacing-7`(3rem) according to https://github.com/opengovsg/postmangovsg/issues/876

Reply:
Remember to also standardise the spacing for both `WatchTheVideo` and `CreateDemoCampaign` modal also in #987.

Reply:
Addressed in https://github.com/opengovsg/postmangovsg/pull/987/commits/8da08f088d4ef33f1d2e76dc84147f354e6e2c46